### PR TITLE
Testnet deployment

### PR DESCRIPTION
**Pull request by kkirka**
This PR adds BSC testnet deployment addresses and ABIs generaated with hardhat-deploy. It also fixes a small bug in one of the deployment scripts (see the first commit for details)

### CODE REVIEW

**Comment by coreyar** [comment]
> Re: creating `networks/` folder – I strongly object storing the same data in different files in the repo, because at some point the addresses in `deployments/` folder will differ from the addresses in `networks/` folder. This is how people work – someone will forget to update something. If we want to include `networks/` in a package, why can't we autogenerate `networks/` from `deployments/`?

yes I believe that was what Simeone was working on with JQ

**Comment by 0xlucian** [comment]
@coreyar  Ahmm not really.. I saw JQ from @kkirka and was playing with it... but we can do this automation for sure

**Review by 0xlucian — APPROVED** [review]

File: contracts/test/VBep20Harness.sol [hunk 1]
```diff
@@ -7,7 +7,7 @@ import "../VBep20Delegate.sol";
 import "../Governance/AccessControlManager.sol";
 import "./ComptrollerScenario.sol";
 
-contract VBep20Harness is VBep20Immutable {
+contract VBEP20Harness is VBep20Immutable {
```
Comment (kkirka) [inline contracts/test/VBep20Harness.sol:10]: Why though?

Comment (coreyar) [reply contracts/test/VBep20Harness.sol:10]: It resolves a typescript compile error originating from generated typechain that complains filename casing doesn't match imports

**Pull request merged**